ads/_ping_.js

```diff
@@ -22,7 +22,11 @@ import {dev} from '../src/log';
  */
 export function _ping_(global, data) {
   global.document.getElementById('c').textContent = data.ping;
-
+  if (!data.nativeIntersectionObserver) {
+    const nullIO = () => {};
+    nullIO.prototype = null;
```

Comment:
This is an error, arrow functions do not have prototypes. Also, setting a prototype to null doesn't prevent instances from getting a prototype chain, only `Object.create(null)` does that.